src/filters/pages.cjs

```diff
@@ -369,7 +369,9 @@ params:
     note: post type to filter by
 */
 const isHome = (collection) =>
-  collection.filter((post) => post.home === true || post.home === 'feature');
+  collection.filter(
+    (page) => page.data.home === true || page.data.home === 'feature',
```

Comment:
@oluoluoxenfree I think this is the change that was needed to fix this filter, and it's IMO an annoying inconsistency in 11ty. Within the context of a given page, data can be access directly, e.g. `page.home`. But within the context of a _page in a collection_, the data is stored under a named `data` key, e.g. `page.data.home`.

https://www.11ty.dev/docs/collections/#collection-item-data-structure